src/ontology/edits/oeo-physical.omn

```diff
@@ -814,6 +814,9 @@ Class: <http://purl.obolibrary.org/obo/BFO_0000023>
     
 Class: <http://purl.obolibrary.org/obo/BFO_0000024>
 
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A tangential proper part is a fiat object part that conforms to the qualitative spatially representaion of TPP in RCC8."@de
```

Comment:
This definition is wrong here, as BFO_0000024 is `fiat object part`.

Reply:
I think that it's a duplicate which went into the wrong section.
Deleting it, will solve this, I think. Will do so now.
@stap-m edited it in `98a12f`, too. Must have overlooked it.
Thanks for figuring out!